server.js

```diff
@@ -482,10 +491,22 @@ app.post('/request-match', (req, res) => {
         options.visits = Number(req.body.visits);
     }
 
+    // Sanitize input
+    //   Convert 0, "", "false" , null, NaN and undefined to boolean false
+    //   Otherwise, true
+    //
+    //   curl -F is_test=true
+    //
+    if(req.body.is_test === "false")
```

Comment:
Should we reverse this so that the default is to make is_test false? That way operation won't change from what @gcp does now by leaving the flag out completely. Unless I'm confusing myself on how === works. Don't undefined === "false" fail and so it'll be set to true by default?